lib/services/authority.go

```diff
@@ -155,6 +155,14 @@ func (c UserCertParams) Check() error {
 	return nil
 }
 
+// GetTLSCA returns the CertificateAutority's TLS certificate authority from TLS key pairs.
+func GetTLSCA(ca CertAuthority) (*tlsca.CertAuthority, error) {
```

Comment:
Can you make a new constructor in the `tlsca` package itself?
Something like `func FromAuthority(ca types.CertAuthority) (*CertAuthority, error)` and rename `tlsca.New` to `tlsca.FromKeys` maybe?